packages/jest-transform/src/ScriptTransformer.ts

```diff
@@ -390,6 +491,135 @@ export default class ScriptTransformer {
     };
   }
 
+  // TODO: replace third argument with TransformOptions in Jest 26
```

Comment:
we might wanna look into landing all the jest 26 changes first, not sure... Would delay this PR so probably not